sourcemash/frontend/static/js/views/item_card_view.js

```diff
@@ -70,16 +93,20 @@ Sourcemash.Views.ItemCardView = Backbone.View.extend({
             this.model.save({saved: false},
                 {success: this.savedToast});
         } else {
-            this.model.save({saved: true},
-                {success: this.savedToast});
-
-            mixpanel.track("Saved", { "Item Title": this.model.get('title'),
-                                    "Feed Title": this.model.feed.get('title') })
-
-            if (!this.model.feed.get('subscribed')) {
-                this.showSubscribeModal({'source':'saved'});
-            }
-        }
+            if (!this.user.get('id')) {
```

Comment:
yeah, but you can take the url and send that to mixpanel if you're concerned about tracking the source.

Wouldn't it be nice if whenever we add a new login_required feature that we never need to write code for pulling up the register modal again?